src/server.py

```diff
@@ -437,25 +405,13 @@ def run_seta_query():
 @app.route("/data/setadetails/")
 @json_response
 def run_seta_details_query():
-    # TODO: remove inactive when buildbot api queries s/inactive/priority/
-    inactive = sanitize_bool(request.args.get("inactive", 0))
```

Comment:
we will have to port some changes to the alertmanager server first- can you file an issue for Ouija so we can track this?